Print labels with their colors on lab label list

zampierilucas opened the pull request:
This takes the `label.Color` hex and converts it to color in HexToRGB in utils, then uses is to set the background color.

Closes: https://github.com/zaquestion/lab/issues/887

Commit:
Print labels with their colors on lab label list

This takes the label.color hex and converts it to color in HexToRGB in
utils, then uses rgbterm lib to convert rgb to terminal color.

Signed-off-by: Lucas Zampieri <lzampier@redhat.com>

zampierilucas commented:
This is what the labels look like with the change.
![image](https://github.com/zaquestion/lab/assets/25491728/e5f9ee9a-b544-4c2a-a5cb-7bc886420db1)

prarit commented:
I like this change but I think it should be optional.  @zampierilucas I think you should add a config option for this to enable the colors.  That way users can set it in their lab.toml and not worry about it.

Commit:
Made Color an arg

Label colored printing can be enabled by the --color param, or by using
the config file.

Signed-off-by: Lucas Zampieri <lzampier@redhat.com>

zampierilucas commented:
@prarit I've updated my mr with you suggestion.

Commit:
Added color to the example table

title

Signed-off-by: Lucas Zampieri <lzampier@redhat.com>

bmeneg reviewed (approved):
We can make this change generic to every lab command.
Since it's a bigger change, this one is fine wrt the request.

teto commented:
was the CI failure linked to this PR ? else maybe it's mergeable ?
